App.jsx

```diff
@@ -1,23 +1,59 @@
-import { StatusBar } from 'expo-status-bar';
-import React from 'react';
-import { StyleSheet, Text, View } from 'react-native';
+import React, { useEffect, useState } from 'react';
+import { StyleSheet, Text, FlatList, View } from 'react-native';
+import axios from 'axios';
+import Article from './components/Article';
+import Hero from './components/Hero';
 
 const App = () => {
+  const [articles, setArticles] = useState([]);
+  const fetchArticles = async () => {
+    const response = await axios.get(
+      'https://fakest-newzz.herokuapp.com/api/articles'
+    );
+    setArticles(response.data.articles);
+  };
+
```

Comment:
Question is if we should move this to a module already at this stage

Reply:
We just wanted to display the article list, we can fix it in the next pr if it is necessary?

Reply:
Sounds fair :)

Reply:
yeah! :)